eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -118,10 +118,12 @@ jobs:
         mv "dotnet-$(Build.SourceVersion)" "$(sourcesPath)"
       displayName: Export VMR sources
       workingDirectory: $(Build.StagingDirectory)
+      condition: and(succeeded(), ne(variables['disableVmrBuild'], 'true'))
```

Comment:
Having to add this condition to every step in vmr-build except the sync related steps feels like a source of regressions.  Given this variable is not utilized in normal CI these regressions won't get detected until we need to disable the build which is usually in a "crunch" moment.  Would it be reasonable to add this condition in the .vsts-ci.yml pipeline in conjunction with running the `Synchronise VMR` stage if disabled?

Reply:
Yeah, if we're fine with having to make a code change to disable this, we can have a parameter inside the `vsts-ci.yml`